### PR TITLE
Problem 1: Remove Nth Node From List

### PR DESCRIPTION
**Pull request by huyn8**
1. Problem 1
2. Solution 
3. Driver
4. An image that goes with the solution

### CODE REVIEW

File: 2020-02-Spring/2_linked_lists/README.md [hunk 1]
```diff
@@ -94,31 +133,64 @@ Go to [Solution](#s3)   [Top](#top)
...
 <details>
 <summary>Click to see solution</summary>
 
-TODO put your solution here :bug:
+### One pass algorithm
 
-</details>
+**Algorithm:**
 
+Instead of using one pointer, we could use two pointers. The first pointer advances the list by n+1 steps from the beginning, while the second pointer starts from the beginning of the list. Now, both pointers are exactly separated by n nodes apart. We maintain this constant gap by advancing both pointers together until the first pointer arrives past the last node. The second pointer will be pointing at the n-th node counting from the last. We relink the next pointer of the node referenced by the second pointer to point to the node's next next node:
```
Comment (hollowsunsets) [inline 2020-02-Spring/2_linked_lists/README.md:149]: Is using one pointer the naive solution for this problem?

Comment (huyn8) [reply 2020-02-Spring/2_linked_lists/README.md:149]: The naive solution could be anything that might not use the two pointers approach. Let me rephrase this so it's not confusing. Thanks.

File: 2020-02-Spring/2_linked_lists/README.md [hunk 2]
```diff
@@ -94,31 +133,64 @@ Go to [Solution](#s3)   [Top](#top)
...
 <details>
 <summary>Click to see solution</summary>
 
-TODO put your solution here :bug:
+### One pass algorithm
 
-</details>
+**Algorithm:**
 
+We will use the two pointers approach for this problem. The first pointer advances the list by n+1 steps from the beginning, while the second pointer starts from the beginning of the list. Now, both pointers are exactly separated by n nodes apart. We maintain this constant gap by advancing both pointers together until the first pointer arrives past the last node. The second pointer will be pointing at the n-th node counting from the last. We relink the next pointer of the node referenced by the second pointer to point to the node's next next node:
```
Comment (mduw) [inline 2020-02-Spring/2_linked_lists/README.md:149]: very clear explanation! looks good to me
